src/css/components/_nav.scss

```diff
@@ -89,6 +89,9 @@ header nav {
 
 @media #{$breakpoint-es}{
   header nav.es-nav {
-    font-size: $sm-es; 
+    font-size: $sm-es;
+    ul li{
```

Comment:
nit pick: space between `li` and `{`. It might be worth adding hound on this repo so that someone doesn't have to comment on syntax :man_shrugging: